File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/kma/json/midlandfcstresponse/MidLandFcstRoot.java
package com.lifedawn.bestweather.data.remote.retrofit.responses.kma.json.midlandfcstresponse;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MidLandFcstRoot {
	@Expose
	@SerializedName("response")
	private MidLandFcstResponse response;
	
	public void setResponse(MidLandFcstResponse response) {
		this.response = response;
	}
	
	public MidLandFcstResponse getResponse() {
		return response;
	}
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/kma/json/midlandfcstresponse/MidLandFcstRoot.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.kma.json.midlandfcstresponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MidLandFcstRoot {
    @Expose @SerializedName("response") var response: MidLandFcstResponse? = null
}